lighthouse-core/audits/dobetterweb/no-old-flexbox.js

```diff
@@ -53,12 +53,12 @@ class NoOldFlexboxAudit extends Audit {
     }
 
     // TODO: consider usage of vendor prefixes
```

Comment:
Since you've expanded the style helper, it wouldn't be too hard to also include the vendor prefixes now. WDYT?